rintoj/silomobile
# feat: add fastlane release scripts

## Description

This PR adds fastlane release scripts for generating and publishing release builds for ios and android via fastlane

## Change log

- Added fastlane config for android project
- Added release scripts

## How to test

Below commands should work properly

```bash
yarn cli release android dev
```
```bash
yarn cli release android prod
```


TODO:

- version number is currently hardcoded to `v0.1.0`
- configure github api to fetch releases and auto increment versions
